src/staging/src/components/header.js

```diff
@@ -1,11 +1,13 @@
 import React from "react"
-import { css } from "@emotion/react"
-import { useStaticQuery, Link, graphql } from "gatsby"
+import MegaMenu from "./megaMenu.js"
```

Comment:
`".megaMenu.js"` filename is case sensitive

Reply:
Case sensitivity, warnings, and React errors have been fixed. I have went ahead and fixed most of the console warnings in and outside of my body of work but there a couple left:

- \components\park\mapLocation.js
- \components\park\parkHeader.js
- \pages\park-search.js

@siva-ka is working in this area and I do not want to step on her toes so I will leave that to her, or whomever picks up the Jira ticket about cleaning up errors/warnings.